### PR TITLE
fix: update deps and fix worker tests

### PR DESCRIPTION
**Pull request by thesayyn**
closes #223

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -143,6 +143,8 @@ jobs:
                   cp -f $GITHUB_WORKSPACE/.github/workflows/ci.bazelrc .bazelrc.test
                   echo "build --config=${{ matrix.config }}" >> .bazelrc.test
                   echo "build --disk_cache=" >> .bazelrc.test
+                  bazel clean
+                  ./test.sh
```
Comment (thesayyn) [inline .github/workflows/ci.yaml:147]: Running tests twice will give us the confidence that bazel clean does not hide potential state bugs.

**Review by gregmagolan — APPROVED** [review]

File: ts/repositories.bzl [hunk 2]
```diff
@@ -40,20 +40,17 @@ def rules_ts_dependencies(ts_version_from = None, ts_version = None, ts_integrit
...
 
-    # TODO: figure out why updating to 1.6.4 breaks the e2e/worker tests; possibly related to the
-    # fs patches caching perf improvements
     http_archive(
         name = "aspect_rules_js",
-        sha256 = "d8eabcd1e05d93147505ea806fa21089926b771d8813f01b92af5dec36617033",
-        strip_prefix = "rules_js-1.6.3",
-        url = "https://github.com/aspect-build/rules_js/archive/refs/tags/v1.6.3.tar.gz",
+        sha256 = "c3b5fd40ec19f3260094321380169abe86dd89e3506c4e44a515a50c1626629b",
+        strip_prefix = "rules_js-1.6.6",
```
Comment (gregmagolan) [inline ts/repositories.bzl:52]: there is a 1.6.7 out now

Comment (gregmagolan) [reply ts/repositories.bzl:52]: can you also bump aspect_bazel_lib below to latest?

Comment (gregmagolan) [reply ts/repositories.bzl:52]: https://github.com/aspect-build/bazel-lib/releases/tag/v1.16.0

**Pull request merged**